### ISSUE TITLE
libdap 3.19.1 source checksum fails

### ISSUE THREAD

**Issue by hajgato** [easybuilders/easybuild-easyconfigs#5471]
Just tried to build `libdap 3.19.1` (PR #5384, from 2 weeks ago), and the checksum of the source fails.
```
== 2017-12-05 21:57:03,092 filetools.py:700 DEBUG Computed sha256 checksum for /apps/brussel/sources/l/libdap/libdap-3.19.1.tar.gz: fb7014b6047cf3fa47d05c0faf258636f664c5fc232ff0886a78dfc5aae29f8f (correct checksum: 5215434bacf385ba3f7445494ce400a5ade3995533d8d38bb97fcef1478ad33e)
```
@boegel Thughts?

**Comment by boegel** [comment]
Here's what I have:

```
$ sha256sum /prefix/sources/l/libdap/libdap-3.19.1.tar.gz
5215434bacf385ba3f7445494ce400a5ade3995533d8d38bb97fcef1478ad33e  /prefix/sources/l/libdap/libdap-3.19.1.tar.gz
[22:23:05] vsc40023@node2001:~/data $ ls -l /prefix/sources/l/libdap/libdap-3.19.1.tar.gz
-rw-rw-r-- 1 vsc40023 vsc40023 2512834 Nov 21 21:29 /prefix/sources/l/libdap/libdap-3.19.1.tar.gz
```

If I re-download (using `--force --sourcepath /tmp/$USER`), the checksum indeed fails for me too:

```
$ sha256sum /tmp/vsc40023/l/libdap/libdap-3.19.1.tar.gz
fb7014b6047cf3fa47d05c0faf258636f664c5fc232ff0886a78dfc5aae29f8f  /tmp/vsc40023/l/libdap/libdap-3.19.1.tar.gz
$ ls -l !$
ls -l /tmp/vsc40023/l/libdap/libdap-3.19.1.tar.gz
-rw-rw-r-- 1 vsc40023 vsc40023 2493230 Dec  5 22:26 /tmp/vsc40023/l/libdap/libdap-3.19.1.tar.gz
```

There's little else we can do other than:

* issue a PR to fix the checksum
* reach out to the developers and ask them wtf is going on (and ask them to please stop doing it)

**Comment by hajgato** [comment]
@boegel you did the PR 14 days ago, and I checked the date of the source file and it is from 3rd December. It seems that they changed the source after you did the easyconfig.

**Comment by boegel** [comment]
@hajgato Indeed, so there's little else we can do than i) update the checksum, ii) ask them to stop doing in-place updates.

**Comment by hajgato** [comment]
@boegel will do the PR.

**Comment by boegel** [comment]
fixed in #5473

**Comment by zao** [comment]
Checksum is reverted to `5215434bacf385ba3f7445494ce400a5ade3995533d8d38bb97fcef1478ad33e` upstream now.
Of course, the website doesn't even _have_ a `3.19.1` page.

**Comment by boegel** [comment]
@zao Fixed in #6305
